packages/url_launcher/url_launcher/README.md

```diff
@@ -65,18 +70,25 @@ on Android 11 (API 30) or higher. A `<queries>`
 element must be added to your manifest as a child of the root element.
 
 Example:
+
+<?code-excerpt "../../android/app/src/main/AndroidManifest.xml (android-queries)"?>
 ``` xml
+<!-- Provide required visibility configuration for API level 30 and above -->
 <queries>
-  <!-- If your app checks for SMS support -->
   <intent>
     <action android:name="android.intent.action.VIEW" />
-    <data android:scheme="sms" />
+    <data android:scheme="https" />
```

Comment:
We should excerpt the `https` block out (by making it a non-contiguous region); https is in the example because it's thing we can use in integration tests on any device that should return true, but there's basically no reason any actual app should need to check for it (for the same reason). I actually recently removed this from the README example.

Reply:
Done, I've removed the `https` block from the docregion, and used an empty "plaster" so it doesn't even shows up as "..." in the docs.